tests/transport/rtps_direct_write/TestMsg.h

```diff
@@ -0,0 +1,72 @@
+#include "dds/DCPS/Serializer.h"
+#include "dds/DCPS/TypeSupportImpl.h"
+
+using namespace OpenDDS::DCPS;
+
+struct TestMsg {
+  ACE_CDR::ULong key;
+  TAO::String_Manager value;
+  TestMsg() : key(0), value("") {}
+  TestMsg(ACE_CDR::ULong msgKey, TAO::String_Manager msgValue)
+    : key(msgKey), value(msgValue) {}
+};
+
+void gen_find_size(const TestMsg& stru, size_t& size, size_t& padding)
+{
+  if ((size + padding) % 4) {
+    padding += 4 - ((size + padding) % 4);
+  }
+  size += gen_max_marshaled_size(stru.key);
+  find_size_ulong(size, padding);
+  size += ACE_OS::strlen(stru.value) + 1;
+}
+
+bool operator<<(Serializer& strm, const TestMsg& stru)
+{
+  return (strm << stru.key) && (strm << stru.value);
+}
+
+bool operator>>(Serializer& strm, TestMsg& stru)
+{
+  return (strm >> stru.key) && (strm >> stru.value.out());
+}
+
+size_t gen_max_marshaled_size(const TestMsg& stru, bool align)
+{
+  ACE_UNUSED_ARG(stru);
```

Comment:
This macro is used for generated code and when different lines are skipped/included by the preprocessor.  It's not needed here.